.github/workflows/ci.yml

```diff
@@ -236,3 +236,40 @@ jobs:
           # happens very frequently on the `macos-runner`, but we haven't seen it on any others).
           cargo run --features github-token-auth --target ${{ matrix.target }} -p spacetimedb-update -- self-install --root-dir="${ROOT_DIR}" --yes
           "${ROOT_DIR}"/spacetime --root-dir="${ROOT_DIR}" help
+
+  cli_docs:
```

Comment:
this was moved verbatim from the `pr-only-ci.yml` file. That file has now been removed since this CI runs on more than just PRs now. It used to only run on PRs because it would fail if the CLI docs were out of sync with spacetime-docs which would require a separate PR in spacetime-docs, but that's no longer the case.